docs/content/quickstart.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Quickstart Guide
+descriptions: Get started using Porter
+---
+
+## Getting Porter
+
+First make sure Porter is installed.
+Please see the [installation instructions](./install.md) for more info.
+
+## Creating a new installer
+Use the `porter create` command to start a new project:
+```
+mkdir -p my-installer/ && cd my-installer/
+porter create
+```
+
+This will create a file called `porter.yaml` which contains the configuration for your installer. Modify and customize this file for your application's needs.
+
+Here is a very basic `porter.yaml` example:
+```
+name: my-installer
+version: 0.1.0
+description: "this application is extremely important"
+invocationImage: my-dockerhub-user/my-installer:latest
+mixins:
+  - exec
+install:
+  - description: "Install Hello World"
+    exec:
+      command: bash
+      arguments:
+        - -c
+        - echo Hello World
+uninstall:
+  - description: "Uninstall Hello World"
+    exec:
+      command: bash
+      arguments:
+        - -c
+        - echo Goodbye World
+```
+
+## Building a CNAB bundle
+
+The `porter build` command will create a [CNAB-compliant](https://github.com/deislabs/cnab-spec/blob/master/101-bundle-json.md) `bundle.json`, as well as build and push the associated invocation image:
+```
+porter build
+```
+
+Note: Make sure that the `invocationImage` listed in you `porter.yaml`  is a reference that you are able to `docker push` to and that your end-users are able to `docker pull` from.
+
+## Running your installer using Duffle
+
+_Wondering the differences between Duffle and Porter? Please see [this page](./porter-or-duffle.md)._
```

Comment:
Let's remove lines 57 through 65, basically skipping to the duffle install part. 

We are working on wrapping duffle install into porter, but we aren't there yet. 😀

Reply:
I'll update the porter-or-duffle file with our new direction. In summary: porter builds on top of duffle, and uses its libraries. For now we use porter to build the bundle, and duffle to install it.

duffle.json isn't necessary which is why I asked you to skip the rebuilding with duffle. The porter build command creates a CNAB compliant bundle already, which duffle can install without further tweaks.